playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -1231,6 +1233,10 @@ generic_env_config:  &edxapp_generic_env
       LOCATION: "{{ EDXAPP_CACHE_COURSE_STRUCTURE_MEMCACHE }}"
       # Default to two hours
       TIMEOUT: "7200"
+    ora2_storage:
```

Comment:
Technically this is an optional cache that's only used if `ORA2_FILEUPLOAD_BACKEND` is set to 'filesystem'

Reply:
Will it break w/out that?

Reply:
Yea see https://github.com/edx/edx-ora2/blob/master/openassessment/fileupload/backends/filesystem.py#L66, but I couldn't see a way to append this key to the CACHES dict separately in `stanford/playground/group_vars/all` :/